### ISSUE TITLE
A11y bug in accordions

### ISSUE THREAD

**Issue by matthieu-castier** [adeo/mozaic-design-system#754]
# 🐞 Styleguide bug report

### Is this a regression?

 No

### Description

 With the accordion macro, there is a concern of accessibility. Indeed, if there are links in the accordion content, when you navigate with the keyboard (with the tab key), the focus is on the links when they are not visible.

## 🔬 Steps to reproduce the issue

You have to create an accordion with 2 elements. In the content of the first element, you have to put a link and then test the keyboard navigation.

The focus will switch to the first title, then to the link (invisible) and then to the second title.

## 🔥 Exception or Error
<pre><code>

</code></pre>

## 🌍 Your Environment

**Styleguide Version:**
<pre><code>

</code></pre>

**Anything else relevant?**

To fix this issue :

In accordion.scss I added `display: none;` in `&__content`.
I removed `border-bottom: get-border(s) solid $color-font-light;` in `&__content`.
I added `display: block;` in ` &:checked { ~ #{$parent}__content {`

In accordion.ftl : 
I added `<div class="mc-divider-bottom" aria-hidden="true"></div>` after `<div class="${contentClass}">`
